awx/ui_next/src/api/models/InventorySources.js

```diff
@@ -5,7 +5,19 @@ class InventorySources extends LaunchUpdateMixin(Base) {
   constructor(http) {
     super(http);
     this.baseUrl = '/api/v2/inventory_sources/';
+
+    this.allowSyncStart = this.allowSyncStart.bind(this);
+    this.startSyncSource = this.startSyncSource.bind(this);
+  }
+
+  allowSyncStart(sourceId) {
```

Comment:
I'm not sure if we should name these based on what they are specifically doing, or if we should come up with something more based on the conventions we've been doing for the other api fn's.  Something like:

readSourceSyncUpdate
createSourceSyncUpdate